app/scripts/modules/openstack/loadBalancer/details/details.html

```diff
@@ -54,35 +54,29 @@ <h3 select-on-dbl-click>
         <dd>{{loadBalancer.subnetName}}</dd>
         <dt>Network</dt>
         <dd>{{loadBalancer.networkName}}</dd>
-        <dt>Port</dt>
-        <dd>{{loadBalancer.externalPort}}</dd>
-        <dt ng-if="lbLink">Link</dt>
-        <dd ng-if="lbLink">
+        <dt ng-if="lbLinks">Links</dt>
+        <dd ng-repeat="lbLink in lbLinks track by $index">
           <a href="{{lbLink}}" target="_blank">{{lbLink}}</a>
           <copy-to-clipboard
             class="copy-to-clipboard copy-to-clipboard-sm"
             text="{{lbLink}}"
             tool-tip="Copy Link to clipboard">
           </copy-to-clipboard>
         </dd>
+        <dt>Method</dt>
+        <dd>{{loadBalancer.method}}</dd>
```

Comment:
Sorry I missed this in the earlier code review... but what is this? And where is the distribution algorithm displayed?